### DISCUSSION TITLE
[Proposal] Few ideas

### DISCUSSION THREAD

**Discussion by trenie222** [sunflower-land/sunflower-land#3260] [Ideas]
Hi guys I recently bought the island and currently exploring, sinking in the project. It's probably a little bit too fast to suggest anything as a newcomer but I just feel like it, if its crap its crap.

### UI
**Lens option in settings. Turn on/off option.**
Hovering an object gives information. 

### SOCIAL
**Sunflower hall**
Monuments in a form of a Sunflower signed with players lands (click to visit) for huge amount of SFL/resources granted to the Pumpkin Plaza (burned).
**Visiting other islands more accessible.**
In player tab when clicking on someones Bumpkin in the plaza - option to click ID of an land to instantly visit it.
**Gift table**
Basically a drawer/bookstand with 1-3 slots where player can leave out some resources for visitors.
it had to be balanced to the point that these gifts wouldn't destroy the gameplay - preferably small donations of low valuable items but actually meaningful for a new player + limited received gifts daily

### Small events
**Fedora**
Adam's fedora got carried away by the wind - catch it to get small gift.
and again little gifts meaningful for a new player + extremely rare chance to obtain fedora nft for example
**Pirates**
Encountering Outcast Bumpkins between travels. Basically an encounter of pirates/cpt of the pirates that contains additional daily quest/mini game to get out.
Personally I find it quite interesting and it could expand to a future season someday as it can be a wider story behind it. 

### ACHIEVEMENTS
**X visits by other farmers.**
**X cheers for the gifts**
**X catches of fedora**

I might not understand what rules are in this space. Is it game time that matters or onchain activity that is much valuable in eyes of web3 players. If the mechanic is even possible to do or not. If you want me to expand on a particular topic I'd love to do that. I'd love to hear your thoughts tho.

**Comment by trenie222** [comment]
Just realized this season was all about kraken and we even have the pirate guy on the beach so maybe the pirate mini event isn't that fresh or just straight up bad.
Although since noone is responding yet I will keep updating this thread with my ideas.

### Wearables
**Mouse pointers**
When picking up crops, cutting down the tree, mining stones/iron/gold animation of particular act.
Instead of mouse when you hover on the tree little axe pops up/picking up crops with gloves. (i am not sure if its possible to do in game via website)
Different mouse pointer wearables obtained by crafting/achievements.
### Mechanics
**Catch-up**
Level island 5/10+
After player' 2 weeks/month inactivity additional soils, tree, stone for X days. (not sure if possible)
Infographic of what's happening right now.
1-7 day streak give additional items(axe, pick axe, seeds) when collecting daily prize
Free BB for 1st? and 7th day streak.

**Comment by trenie222** [comment]
At first let me tell you a little about my relation with games in general. Personally I don't find much difference between web2 and web3 monetization. Although in web3 its builded, in web2 games selling ing stuff been for years and it's doing more than fine if you ask me. I used to play games for **fun**, **competitively** and for **money**. See, this three reasons can actually mix together. If I find mix of 3, that's perfect game for me. It's also very fluid and that's why I find important for the game to have those 3 in it. What I mean by fluid is, If you don't feel like playing current tittle for fun there is also competition thrive or money case holding onto why I will still play. If I don't feel like playing for money I want to have fun etc. Games like this are very rare tho, atleast in my personal opinion. The reason why I am elaborating on this more is that you possibly understand my stance on the case better and I got the welcoming message that it's not a financial product and didn't want you to portray me as a person that came here **only** for the money. I came here **mainly** for the money. And that's big difference.

### Eco
**Sharing**
From what I understand it's possible to basically share NFTs in the game. If my friend has all vital EXP NFTs he can simply transfer me those for 30 minutes, I eat my stuff and resend. Well my FUD mind went immiediately to the place that players who are abusing this are working against ECO of the game. I am not talking specifically about this case - case of two guys/girls enjoying the game, spending some money, sharing what they have/can to each other to boost their gaming experience. I think that's completely fine tbh. But my mind is going even further telling me that there are guilds in web3 who can actually abuse it. They make $ contribution to the guild > leader buys nfts > they share it between themselves. To be completely honest once I wrote it, it actually makes perfect sense for me - it's effiecient, doesnt brake any rules. Although if there are many groups like that it slows market drastically. Let's say there are 5 groups of 20 people in it. So 100 man are sharing 5 items between eachother. Especially in game like SFL where you can just stack your food endlessly and choose when to use it. It translates to less demand of NFTs>less liquidity>lowering value.

So now; I don't know what's your vision of the game and what's your standing on this topic but I personally would limit this. Atleast a little bit.
My answer would be
**Make renting system(i am not sure if its possible without own marketplace)**
**NFTs placed on island are binded for a week to a player.**
**NFTs with +X%exp have energy per consumed meal, you have to recharge them** (this one is sketchy tho, its hitting normal players and could be a pain in the ass with additional eco calculations)
**Daily quests of eating some kind of food**
**Forbid excessive sharing, make rules about the case** (don't like it)

At the same time I am not sure if that's even a case happening in the game but from my experience it's definetely a thing. Just sharing my thoughts. 
Example of devil wings shared:
![image](https://github.com/sunflower-land/sunflower-land/assets/154707245/8f0c0110-0178-432c-acb7-b58790442c77)
![image](https://github.com/sunflower-land/sunflower-land/assets/154707245/6db099c3-3c76-4e02-a2f8-4ebd568ef9fc)
![image](https://github.com/sunflower-land/sunflower-land/assets/154707245/79315bf9-d71b-4642-ad35-12f787357918)

They've been sharing like that for quite some time


### Expansion/Social/Mechanics
**Guilds, clans or Archipelagos what I would name them**
I can only imagine how many threads in this case was before I joined the community. Although If anyone is interested in my thoughts I would happy to add my two cents.


### UI
Timers of Kitchen, Fire pit, bakery etc. displayed on the buildings or after hovering the building(i think this one is better).

**Comment by trenie222** [comment]
### Community events
**Shaman Pumpkin**
Resident of Pumpkin Plaza after his requirements are met he performs rain dance.
Raining x time giving x bonus to crop growth time/yield.
**Sunflower Sunday**
After week passes and certain hidden requirements are met (can get some tips from npcs possibly) event occur on Sunday 3 times different hours.
Either big Sunflower spawn in center of Plaza for 30 minutes free to harvest every 4 secs or emerges from the see on players' island.
**Help the beaver**
Spawns big ass tree that blocks path to potion house for example. Reward depends on how much do you contribute axes. But basically even smallest contribution pays off. 

To be honest I might be rushing into things that I am not sure even if are possible/needed. I'd appreciate any feedback, thanks in advantage.

### Delivery system
**NPC courier**
NPC that travels from Plaza to your island. Not sure how obtainable, or just simply a visitor on Island every 6h that hangs around for 10-15 minutes and you can hand him resourcers needed to deliver to all NPCs for a small fee.
Not sure why on PC I am having literally 1 FPS when visiting Plaza even tho my hardware acceleration in browser is turned on. It's just pain in the ass trying to deliver stuff for 15minutes and from what I heard I am not the only one having this issue.

**Comment by trenie222** [comment]
### Mechanics
**Beach** 
Remake of 1 tile connected to the sea, adding a small beach/inflood
![image](https://github.com/sunflower-land/sunflower-land/assets/154707245/d6636fc0-7735-48be-9dc9-bed8c7387db6) 
Everyday there are possible few random prizes spotted on the beach every HOUR. These are small amounts of resources and extremely rare chance of higher rewards. Inflow always contain 2 or more prizes.

1. Branches/planks from a sunken ship giving small amount of wood. No tool needed to gather.
2. Rocks by the sea that give small amount of stones. No tool needed to gather. 
3. There are small chances of getting message in a bottle that activates quest/sellable trash object. If quest > one time delivery to Beach NPC that pays 0.625 SFL.
4. Extremely rare treasure chest with gold? like 1-2/other items
New players are starting with couple additional branches and rocks to collect by the get go or beach CD sped up to 15minutes for the first 3/7 days.
Daily limit 20 collected beach items.

I like the idea so much I would expand it further although my first thinking was to help out new players/more clicks and stuff to do to keep person involved. 

**How I would expand it further?**
1. Widening the beach on the island/leveling beach=adding watch tower, floating nets, or more fun/cute way sandcastle on the beach, hammock . Each level pays out in better rewards and later on when leveled more possible prizes at once/enlarging daily limit.
2. Craftable sharpened wooden stick tool that gives chance to catch lowest tier fishes. < mechanic mainly for the new players if they wish to level up faster instead of minig stone for example or they wish to spend money. Daily limit 5 catched fishes or 10 uses.
3. Fish nets. Craftable/placable object after X time, gives random fishes/trash. These are possible to use multiple times unless they are wrecked. Unlocked on later stages of the beach.
3a) SFTs/NFTs that gives you +1 placed fishnet.
5. NFTs/SFTs that improve quality of rewards from inflows, higher chance of 3/4/5 inflows at once, higher chance of obtaining treasure chest.
6. Requirements to collect inflow prizes
Specific NFTs enable you to collect better resources for example Rock Golem enables you to loot solid rock that is available for level 2 beach inflows. Foreman beaver allows you to collect side of the sheep from the inflow. Trident enabling hunting for fishes near the beach for free with higher chances to catch.
**Achievements**
Message in a botttle delivery.
Collect 400/2000/4000 inflows
1 treasure chest looted.
Treasure hunter 10 treasure chest looted.
Caught 100/1000 fishes with the use of fish net.

**Comment by Jericho-rpc** [comment]
Totally agree with you, a lot of players have left this game because the developers are so bad and the players invested a lot of time and money but didn't allow the players to gain anything? The developers' mindless increase in output inflation has made the game unbalanced, causing players who invested in the game to not only waste money and time, but why do they have to complete those boring tasks every day? There is no logic at all. Old players are being lost little by little. Developers should do something. Put away their arrogance and listen to the opinions of the vast majority of investors and players!
